### ISSUE TITLE
Bottom sheet needs focus management

### ISSUE THREAD

**Issue by marcysutton** [angular/material#571]
To be accessible, when a Bottom Sheet component is opened, focus should be sent to its first interactive control ("Share" in the Show As List option). This will vastly improve the experience for keyboard and screen reader users. Currently, [focus is blurred](https://github.com/angular/material/blob/master/src/components/bottomSheet/bottomSheet.js#L145) from the triggering button; this causes focus to be lost, a major accessibility issue.

![Active bottom sheet screenshot](https://cloud.githubusercontent.com/assets/1045233/4962720/9678a1e4-66e6-11e4-9c9f-884613a1cd16.png)
